app/models/petition.rb

```diff
@@ -47,7 +47,7 @@ class Petition < ActiveRecord::Base
 
   facet :all,       -> { not_archived.by_most_recent }
   facet :open,      -> { not_archived.open_state.by_most_popular }
-  facet :rejected,  -> { not_archived.rejected_state.by_most_recent }
+  facet :rejected,  -> { not_archived.rejected_state.or(hidden_state).by_most_recent }
```

Comment:
I'm going to put my trust in the PostgreSQL query planner here and assume that there's no difference in plan between `state IN ('rejected', 'hidden')` and `state = 'rejected' OR state = 'hidden'` 😂